docs/sources/administration/configuration.md

```diff
@@ -976,6 +976,41 @@ Sets the minimum interval between rule evaluations. Default value is `1`.
 
 > **Note.** This setting has precedence over each individual rule frequency. If a rule frequency is lower than this value, then this value is enforced.
 
+# Configures for how long alert annotations should be kept. Default is 0 which keeps them for ever
```

Comment:
See default.ini edits.

Also, please correct the formatting from lines 979 to 985.

Reply:
Done